src/js/core/widget/BaseKeyboardSupport.js

```diff
@@ -1187,7 +1187,7 @@
 				var self = this;
 
 				if (!ns.getConfig("keyboardSupport", false)) {
```

Comment:
I wondering about inversion of this condition and move the code below into this conditional.

Reply:
I cannot verify this patch is solving SVACE issue. How you verify it?

Reply:
I don't know the way to check if this issue is solved or not. Usually next run tells us if we achieved required number of svace issues.
I just tested UIComponets app if there is no regression after this change.

Reply:
Both options are fine to me. I can change the code according to your comment.

Reply:
I prefer do inversion of condition and move the code into the condition block.

I reviewed all commit in this PR. I think this is only thing to fix.

Reply:
Done.